util/helpers.ts

```diff
@@ -73,3 +98,59 @@ export const getTwitterHandle = (url?: string) => {
   const twitterHandle = url?.split('/').pop();
   return twitterHandle;
 };
+
+/**
+ * function that returns day, date and key of a given session
+ * @param schedule ISchedule
+ * @returns Array<dates>
+ * @example
+ */
+export const getDaysFromSchedule = (schedule: ISchedule): Array<IDateForDayButton> => {
+  const keys = Object.keys(schedule.data);
+  const datesToSave = keys.map((key, index) => {
+    const date = new Date(key).getDate();
+    const dateWithSuffix = getSuffixForDate(date);
+    return {
+      day: `Day ${index + 1}`,
+      date: `${date}${dateWithSuffix}`,
+      key,
+    };
+  });
+  return datesToSave;
+};
+
+/**
+ * function that returns suffix for a date
+ * @param date number
+ * @returns Array<dates>
+ * @example 2 returns 2nd
+ */
+const getSuffixForDate = (date: number) => {
+  const suffix =
+    date > 0 ? ['th', 'st', 'nd', 'rd'][(date > 3 && date < 21) || date % 10 > 3 ? 0 : date % 10] : ('' as string);
```

Comment:
The ` as ` assertion is not needed here since `''` is already inferred to be a string.

```ts
const suffix = date > 0 ? ['th', 'st', 'nd', 'rd'][(date > 3 && date < 21) || date % 10 > 3 ? 0 : date % 10] : '';
```